htdocs/public/ticket/list.php

```diff
@@ -374,18 +374,14 @@
 		$sql .= " t.datec,";
 		$sql .= " t.date_read,";
 		$sql .= " t.date_close,";
-		$sql .= " t.tms,";
-		$sql .= " type.label as type_label, category.label as category_label, severity.label as severity_label";
```

Comment:
Line 638,we have 
$obj->type_label
If you remove this, this property will be empty...